samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_connmon.c

```diff
@@ -31,11 +34,16 @@ static void modem_data_update(struct k_work *work)
 		return;
 	}
 
-	/* TODO: Make this less hard-coded */
-	lwm2m_engine_set_s8("4/0/0", bearers[0]);
+	if (modem_param.network.lte_mode.value != 0) {
+		lwm2m_engine_set_s8("4/0/0", LTE_FDD_BEARER);
```

Comment:
Should this be `lwm2m_engine_set_u8()` to align with `bearers` being `uint8_t`?

Reply:
Yes, that would actually be a correct thing to do.